### PR TITLE
feat: add rundeck 3.x installation

### PR DESCRIPTION
**Pull request by weslleycamilo**

### CODE REVIEW

File: tasks/projects.yml [hunk 1]
```diff
@@ -1,13 +1,22 @@
 ---
-- name: Run rd-project if project directory not exists in file system
-  command: "/usr/bin/rd-project -p {{item}} -a create -v"
+- name: Create properties for projects
+  template:
+    src: "project-properties/{{ item }}.properties.j2"
+    dest: "/etc/rundeck/{{ item }}.properties"
```
Comment (weslleycamilo) [inline tasks/projects.yml]: Create new folder for its properties to clean up the `/etc/rundec/` folder

**Review by mtulio — CHANGES_REQUESTED** [review]
@weslleycamilo all of this changes was yours or do you merge changes from upstream role and yours?
Is it possible to separate each one to understand what was your changes (and review it), from community repo.

When we do that, we will understand what and how we can contribute back to the upstream.

**Comment by weslleycamilo** [comment]
It was all mine.  There are some changes which are in our role which we didn't push back to the community so I believe we can merge it and push everything we've done until now to the community so we don't have to wait his approval to get our code updated right ?

**Review by cassiomcouto — APPROVED** [review]

File: templates/nginx/rundeck.conf.j2 [hunk 2]
```diff
@@ -52,3 +41,12 @@ server {
         proxy_pass http://127.0.0.1:{{ framework_server_port }};
     }
 }
+
+#server {
```
Comment (raphapr) [inline templates/nginx/rundeck.conf.j2:45]: what about remove these commented lines? once there is no use for this

File: templates/rundeckd.j2 [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This file will be sourced before the /etc/rundeck/profile,
+# allowing your exports to take precedence.
+
+RUNDECK_TEMPDIR={{ rundeck_tmpdir }}
+
+RDECK_JVM_SETTINGS="${RDECK_JVM_SETTINGS:- -Xmx{{max_heap}}m -Xms256m -XX:MaxMetaspaceSize=256m -server}"
```
Comment (raphapr) [inline templates/rundeckd.j2]: where `max_heap` is defined? what about parametrize jvm settings as default vars instead? it gives more versatility for the role

please, see my sugestions and let me know what you think about.

Comment (raphapr) [inline templates/rundeckd.j2]:
```suggestion
RDECK_JVM_SETTINGS="${RDECK_JVM_SETTINGS:- {{ rundeck_jvm_settings }}"
```

File: templates/rundeckd.j2 [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+RDECK_JVM_OPTS="-Drundeck.jaaslogin=true \
```
Comment (raphapr) [inline templates/rundeckd.j2:8]:
```suggestion
RDECK_JVM_OPTS="{{ rundeck_jvm_opts }}"
```

File: templates/rundeckd.j2 [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+       -Djava.security.auth.login.config=/etc/rundeck/jaas-multiauth.conf \
```
Comment (raphapr) [inline templates/rundeckd.j2:9]:
```suggestion
```

File: templates/rundeckd.j2 [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+       -Dloginmodule.name=multiauth"
```
Comment (raphapr) [inline templates/rundeckd.j2]:
```suggestion
```

File: defaults/main.yml [hunk 7]
```diff
@@ -56,10 +55,6 @@ rd_create_projects: []
 # Users list
 rd_create_users: []
 
-# Hipchat configuration
-hipchat_token: null
-hipchat_template: /etc/rundeck/hipchat_default.ftl
-
 # Rundeck TMPDIR
 rundeck_tmpdir: /tmp/rundeck
```
Comment (raphapr) [inline defaults/main.yml:60]:
```suggestion
rundeck_jvm_settings: "-Xmx256m -Xms256m -XX:MaxMetaspaceSize=256m -server"
rundeck_jvm_opts: "-Drundeck.jaaslogin=true -Djava.security.auth.login.config=/etc/rundeck/jaas-multiauth.conf -Dloginmodule.name=multiauth"
```

**Review by raphapr — CHANGES_REQUESTED** [review]

File: templates/rundeckd.j2 (hunk 3 above)
Comment (weslleycamilo) [reply templates/rundeckd.j2]: great @raphapr

File: templates/nginx/rundeck.conf.j2 (hunk 2 above)
Comment (weslleycamilo) [reply templates/nginx/rundeck.conf.j2:45]: I'm going to still use it as the old rundeck but just to not use the certs I commented it at the time.

**Review by raphapr — APPROVED** [review]

**Review by raphapr — APPROVED** [review]

File: defaults/main.yml [hunk 8]
```diff
@@ -56,13 +55,10 @@ rd_create_projects: []
...
```
Comment (raphapr) [inline defaults/main.yml:59]:
```suggestion
rundeck_tmpdir: /tmp/rundeck

# JVM parameters
```

File: defaults/main.yml [hunk 9]
```diff
@@ -56,13 +55,10 @@ rd_create_projects: []
...
-
+rundeck_jvm_settings: "-Xmx256m -Xms256m -XX:MaxMetaspaceSize=256m -server"
+rundeck_jvm_opts: "-Drundeck.jaaslogin=true -Djava.security.auth.login.config=/etc/rundeck/jaas-multiauth.conf -Dloginmodule.name=multiauth"
```
Comment (raphapr) [inline defaults/main.yml:63]:
```suggestion
rundeck_jvm_opts: "-Drundeck.jaaslogin=true -Djava.security.auth.login.config=/etc/rundeck/jaas-multiauth.conf -Dloginmodule.name=multiauth"

```

**Pull request merged**